### DISCUSSION TITLE
ΜΕΡΟΣ Β - ΕΡΩΤΗΜΑ 3

### DISCUSSION THREAD

**Discussion by Katethoma** [sdy60/syllabus#45] [Q&A] [answered]
Καλησπέρα. Ήθελα να ρωτήσω αν για την απάντηση στο ερώτημα 3 του ΜΕΡΟΥΣ Β μπορούμε να χρησιμοποιήσουμε την εφαρμογή Figma ή μόνο την προτεινόμενη

**Comment by sotkot** [comment, marked as answer]
Καλησπέρα Κατερίνα,

Νομίζω ότι ο καθηγητής έχει στείλει ένα URL με προτεινόμενα εργαλεία, χωρίς να σημαίνει ότι αυτά είναι δεσμευτικά. Οπότε νομίζω ότι το Figma είναι οκ σαν επιλογή.

**Reply by Katethoma** [reply]
Ευχαριστώ !
